src/controllers/auth.controller.js

```diff
@@ -0,0 +1,32 @@
+const authService = require("../services/auth.service");
+
+const signUp = async (req, res) => {
+  try {
+    const { name, birthdate, phoneNumber, gender, address, email, password } =
+      req.body;
+    await authService.signUp(
+      name,
+      birthdate,
+      phoneNumber,
+      gender,
+      address,
+      email,
+      password
+    );
+    res.status(201).json({ message: "success" });
+  } catch (err) {
+    res.status(err.statusCode || 400).json({ message: err.message });
+  }
+};
+
+const signIn = async (req, res) => {
+  try {
+    const { email, password } = req.body;
+    const accessToken = await authService.signIn(email, password);
+    res.status(200).json({ accessToken: accessToken });
+  } catch (err) {
+    res.status(err.statusCode || 401).json({ message: err.message });
+  }
+};
```

Comment:
해당 부분도 마찬가지입니다.
아래 예시 코드를 참고하여 적용해주세요!

```suggestion
const signIn = async (req, res) => {
  try {
    const { email, password } = req.body;
   
   if(!email || !password) {
     throw new Error("key error");
   }
   
    const accessToken = await authService.signIn(email, password);
    
    res.status(200).json({ accessToken: accessToken });
  } catch (err) {
    res.status(err.statusCode || 401).json({ message: err.message });
  }
};
```

Reply:
꼼꼼하게 리뷰 해주셔서 더 많이 배우는 것 같습니다~!